tools/assign_missing_instruments.php

```diff
@@ -58,6 +58,7 @@
 $query = "SELECT ID, subprojectID from session";
 if (!empty($argv[1]) && $argv[1]!="confirm") {
     $query .=" WHERE visit_label='$argv[1]'";
```

Comment:
is this query line required? It seems like it's done with a prepared statement below.